app/cafe.py

```diff
@@ -0,0 +1,19 @@
+from app.errors import NotVaccinatedError, \
+                        OutdatedVaccineError, \
+                        NotWearingMaskError
+import datetime
```

Comment:
imports should be organized according to a PEP 8  gudeline:
https://peps.python.org/pep-0008/#imports

I mean the proper way is:

    Standard library imports.
    Related third party imports.
    Local application/library specific imports.

You should put a blank line between each group of imports.

Also multiple imports better to be organized according to checklist
https://github.com/NEkropo1/cafe-test-task-pt/blob/master/checklist.md